percona/controller/pgcluster/controller.go

```diff
@@ -535,6 +539,47 @@ func (r *PGClusterReconciler) reconcileCustomExtensions(cr *v2.PerconaPGCluster)
 		extensionKeys = append(extensionKeys, key)
 	}
 
+	if cr.CompareVersion("2.6.0") >= 0 {
+		var extensionKeysForDeletion []string
+		var extensionsKeysAnnotations []string
```

Comment:
This name is confusing, it reads as it a list of annotations rather then list of custom extensions from the annotation.

You could simply name it as `installedExtensions`, and for `extensionKeysForDeletion` you can name it `removedExtension`.